Philips.CodeAnalysis.Test/Maintainability/Maintainability/AvoidInvocationAsArgumentTest.cs

```diff
@@ -250,6 +250,37 @@ public void MyTest()
 			VerifyFix(errorContent, fixedContent);
 		}
 
+		[TestMethod]
+		public void AvoidInvocationAsStaticMemberAccessArgumentFixTest()
+		{
+			string errorContent = @"
+class Foo
+{
+  public static string Do() { return ""hi"";}
+  public static void Moo(string x) { }
```

Comment:
On Line 260, change 'x' to 'value'. Then do the same on lines 271. For 274 and 275, change 'x' to 'resultOfDo'. That should get the code coverage up.

Reply:
Done